webrender/res/prim_shared.glsl

```diff
@@ -74,6 +74,9 @@ in int aLayerIndex;
 in int aElementIndex;
 in ivec2 aUserData;
 
+// get_fetch_uv is a macro to workaround a macOS Intel driver parsing bug.
```

Comment:
I'd add `//TODO: convert back to a function once the driver issues are resolved, if ever`
Also, "workaround" here should be "work around" for a verb is expected.